### DISCUSSION TITLE
Cannot get other servers to show on Plan

### DISCUSSION THREAD

**Discussion by Hellmanx** [plan-player-analytics/Plan#3051] [Q&A] [answered]
So far, one server seems to be working fine.  However, if I try to add another server, it will not be recognized.

Both configs are near identical with two different things changed:
ServerName
Disable_Webserver

Both servers have different names and only one server is hosting the webserver.

**Comment by AuroraLS3** [comment]
Double check instructions in step 7 https://github.com/plan-player-analytics/Plan/wiki/Bungee-Set-Up#step-7-set-up-plan-on-game-servers

**Reply by Hellmanx** [reply, marked as answer]
I found it, it's under this drop menu.  I guess you the server tab is removed and replaced with this now?
Not a fan, but it is what it is.
![image](https://github.com/plan-player-analytics/Plan/assets/51136027/2201def6-b56b-4151-ba2f-5add2332dfba)
